contracts/upgradeable_contracts/native_to_erc20/HomeBridgeNativeToErc.sol

```diff
@@ -121,26 +126,43 @@ contract HomeBridgeNativeToErc is EternalStorage, BasicHomeBridge, RewardableHom
         }
     }
 
-    function onExecuteAffirmation(address _recipient, uint256 _value, bytes32 txHash) internal returns (bool) {
+    /**
+     * @dev Internal callback to be called on successfull message execution.
+     * Should be called only after enough affirmations from the validators are already collected.
+     * @param _recipient address of the receiver where the new coins should be unlocked.
+     * @param _value amount of coins to unlock.
+     * @param _txHash reference transaction hash on the Foreign side of the bridge which cause this operation.
+     * @param _hashMsg unique identifier of the particular bridge operation.
```

Comment:
Please add that it is not used within this method but is required for unification with other bridge modes.

Reply:
Done.